### ISSUE TITLE
Translate Portal into Spanish

### ISSUE THREAD

**Issue by lalitha-jonnalagadda** [18F/crt-portal#202]
## Narrative
As a Spanish speaking user, I want read the form in Spanish so that I can submit my civil rights concern. 

## Background
This is one of our launch goals

## Assumptions
- CRT only views don't need to be translated
- We will probably also need a language picker icon with user interaction that is incorporated into the site header but that will be a subsequent task.

## Acceptance criteria
- [ ] All form text is translated to Spanish
- [ ] If browser settings are set to Spanish, the Spanish site will be served to the user

## Technical Notes
.po file with Spanish translation: https://drive.google.com/drive/folders/1PG_rrUFkBGx5kF3D76HQQw57ivzB4cax
- [ ]proof the .po for formatting in python looking for stray quotes or tags etc. (Lindsay started this but we can hand off if someone else gets to this ticket first.)
- [ ]compile file
- [ ]set up/troubleshoot language settings for the app

**Comment by LindsayYoung** [comment]
I think we can request a translation of the form using this form, I don't see where to submit it but I would try eeo@gsa.gov first https://insite.gsa.gov/cdnstatic/insite/Request_for_Translation_or_Interpretation_Services.pdf

**Comment by LindsayYoung** [comment]
We are ready for this card!

**Comment by LindsayYoung** [comment]
Remember to include @Jacklynn's update: https://github.com/usdoj-crt/crt-portal/pull/417

**Comment by lalitha-jonnalagadda** [comment]
Translated and updated file from Marta and team : [https://app.zenhub.com/files/183258764/8f3e146d-645a-4e5e-b9ef-2ed79c0f8f63/download](https://app.zenhub.com/files/183258764/8f3e146d-645a-4e5e-b9ef-2ed79c0f8f63/download)

**Comment by Jkrzy** [comment]
https://github.com/usdoj-crt/crt-portal/commit/7d1d9ab9ce53e59ad0cbb3ee4dfbac4cc2371e48 contains the required syntax fixes for the received filed

**Comment by Jkrzy** [comment]
Initial draft PR here: https://github.com/usdoj-crt/crt-portal/pull/432.

I think this PR represents an MVP for this card, it gets us to compiled translation messaging, the setup required on the backend in Django, and a temporary input to select language rendered in the footer of the page.

There are some translation fixes/tweaks that need to be done, like the banner and footers are not translated at the moment. Should I start tackling those here or will be handle in another issue?

**Comment by LindsayYoung** [comment]
Just merged the first round of translation work!
- You will get the spanish site if your browser language is set to Spanish!
- You can use the language selector at the end of the page to switch languages!

To do:
- We found a few places where we need to get some more translations
- Add back the line that connects the step bubbles, that was causing issues, so we wanted to tackle that in parallel

**Comment by LindsayYoung** [comment]
@Jkrzy, it looks like we lost some the the translations after merging in some PRs, most notably the heading.  We probably just need another command in our buildchain to make sure we are generating all the right files.

**Comment by LindsayYoung** [comment]
Remaining edits are tracked in a separate issue here: https://app.zenhub.com/workspaces/doj-crt-intake-scrum-board-5d03bf56c1c8a35d482eca0f/issues/18f/crt-portal/467
